server/server.js

```diff
@@ -22,11 +23,34 @@ const STATIC_PATH = path.join(__dirname, '../public');
 
 const app = express();
 
+function allLangs() {
+  return fs
+    .readdirSync(path.join(STATIC_PATH, 'locales'))
+    .map(function(f) {
+      return f.split('.')[0];
+    })
+    .join(',');
+}
+
+function prodLangs() {
+  return require(path.join(
+    __dirname,
+    '..',
+    'package.json'
+  )).availableLanguages.join(',');
+}
+
+const availableLanguages = conf.l10n_dev ? allLangs() : prodLangs();
+
 app.engine(
   'handlebars',
   exphbs({
     defaultLayout: 'main',
-    partialsDir: 'views/partials/'
+    partialsDir: 'views/partials/',
+    helpers: {
+      availableLanguages,
+      l10nDev: conf.l10n_dev
```

Comment:
No, I was going to use it to add pontoon.js but the CSP and other changes scared me away for now